docs/source/key_components/rl_toolkit.rst

```diff
@@ -187,6 +220,23 @@ for sampling purposes. An ``ExperienceMemory`` is a storage facility for experie
 a policy for storing and retrieving training data. Sampling from the experience memory can be customized by 
 registering a user-defined sampler to it.  
 
+``ExperienceSet`` offers a list-like usage:
```

Comment:
We don't have ``ExperienceSet`` any more, do we?

Reply:
Yes, we don't have it any longer since `Buffer` is merged into policies/algorithms.